app/utils/promisify.js

```diff
@@ -0,0 +1 @@
+export default (fn) => new Promise((resolve) => fn((result) => resolve(result)));
```

Comment:
You are right it's a left over of the rebase, going to remove it. Thanks.
On Sat, 11 Jul 2015 at 15:25 Dmitry Polushkin <notifications@github.com>
wrote:

> In app/utils/promisify.js
> <https://github.com/iam4x/isomorphic-flux-boilerplate/pull/69#discussion_r34412605>
> :
>
> > @@ -0,0 +1 @@
> > +export default (fn) => new Promise((resolve) => fn((result) => resolve(result)));
>
> Is it used yet anywhere?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/iam4x/isomorphic-flux-boilerplate/pull/69/files#r34412605>
> .
>